ward/_run.py

```diff
@@ -139,7 +140,8 @@ def run(ctx: click.Context):
 @click.option(
     "--show-diff-symbols/--hide-diff-symbols",
     default=False,
-    help="If enabled, diffs will use symbols such as '?', '-', '+' and '^' instead of colours to highlight differences.",
+    help="If enabled, diffs will use symbols such as '?', '-', '+' and '^' "
+    "instead of colours to highlight differences.",
```

Comment:
I prefer using multi-line strings `"""` instead of implicit string concatenation for these, especially in cases where the framework will automatically handle dedenting, which `click` does.

Reply:
I've updated this to use a multi-line string :)